---
name: "Oliver Stegle"
role: "German Cancer Research Center (DKFZ); European Molecular Biology Laboratory (EMBL)"
subtitle: "From genotype to phenotype with single-cell resolution"
image: "assets/img/keynote/stegle.jpg"
social:
  - url: https://twitter.com/statgenomics
    icon: fab fa-twitter
  - url: https://linkedin.com/in/oliver-stegle-4b128818
    icon: fab fa-linkedin-in
---
<!--
This Space here will be used to explain what the keynote talk will be about.

{:.list-inline}
- When: Monday 00 October 2020 @11:00AM
- Where: Zoom?
-->
<!--
Division Computational Genomics and Systems Genetics
German Cancer Research Center (DKFZ)
Im Neuenheimer Feld 280
69120 Heidelberg, Germany
Web: https://www.dkfz.de/en/bioinformatik-genomik-systemgenetik
Phone: +49 6221 42-3598

Research Group Leader, Genome Biology Unit
European Molecular Biology Laboratory (EMBL)
MeyerhofstraÃŸe 1
69117 Heidelberg, Germany
Web: www.embl.de/research/units/genome_biology/stegle
-->
About the talk: The study of genetic effects on gene expression and other molecular traits using bulk sequencing has allowed for the functional annotation of disease variants in diverse human tissues. Advances in single-cell RNA sequencing and multi-omics protocols provide for unprecedented opportunities to greatly increase the resolution of such genetic analyses, allowing to assess gene regulatory effects at the resolution of cell types, cell states and even in individual cells in human tissues. In this talk, I will present computational strategies for analyzing and integrating population-scale multi-omics dataset. I will then describe applications of these strategies to population-scale single-cell sequencing dataset from genetically diverse human iPSCs across differentiation towards a neuronal fate. Our data provide unprecedented opportunities to map regulatory variants both in major cell types but also in subtle subtypes and across cellular differentiation, revealing dynamic changes of regulatory variants. We describe novel disease-relevant linkages of several of the regulatory variants identified, thereby illustrating how this technology can open novel opportunities to study risk factors for human diseases.
